adapters/info.go

```diff
@@ -37,6 +37,11 @@ type InfoAwareBidder struct {
 
 func (i *InfoAwareBidder) MakeRequests(request *openrtb.BidRequest, reqInfo *ExtraRequestInfo) ([]*RequestData, []error) {
 	var allowedMediaTypes parsedSupports
+
+	if len(request.Imp) == 0 {
+		return nil, []error{BadInput("Empty Imp array in bid request")}
+	}
```

Comment:
The true branch of this if statement is unreachable because request validation in `endpoints/openrtb2/*auction.go` discards requests with empty `[]Imp`. Nevertheless, I believe having it in place makes for a more robust function IMO. What do you think?

Reply:
This `info` bidder is a special case. (.. and kind of gross? when wouldn't we have an info aware bidder? why not just build this into the core?.. all questions for another time).

If this code is unreachable, I vote to remove it.

Reply:
Corrected it so we can exit with an error without the need of this unreachable code. Let me know what you think of the new logic.

Removed